hoot-core-test/src/test/cpp/hoot/core/visitors/TransliterateNameVisitorTest.cpp

```diff
@@ -57,9 +57,9 @@ class TransliterateNameVisitorTest : public HootTestFixture
     TransliterateNameVisitor v;
     map->visitRw(v);
     QStringList names = n1->getTags().getNames();
-    CPPUNIT_ASSERT_EQUAL(true, TransliterateNameVisitor::isLatin(names[0]));
-    //characters is latin, no "note, Transliterated Name" added.
-    CPPUNIT_ASSERT_EQUAL(true, n1->getTags().find("note") == n1->getTags().end());
+    CPPUNIT_ASSERT_EQUAL(false, TransliterateNameVisitor::isLatin(names[0]));
+    //characters is not latin, "note, Transliterated Name" added.
+    CPPUNIT_ASSERT_EQUAL(false, n1->getTags().find("note") == n1->getTags().end());
```

Comment:
Has this test been broken for a while?  It looks like it might have been based on the text you push in and the test below (we generally try to test both true and false inputs) but why has `TransliterateNameVisitor::isLatin` different now?

Reply:
If any character in the string passed to isLatin() is not Latin it is supposed to return false, using Qt's QChar::toLatin1() method.
The text أَلِف clearly has non-latin characters but Qt4 accepted it while Qt5 returns 0 and isLatin() fails as I would expect.
The initial implementation of this test by @bwitham checked for Qt's result of this being a Latin text. Maybe there is more to it I'm not seeing, but I figured the reverse is the desired behavior.

Reply:
Makes sense to me; I just don't know why it was written the way it was before.  I good with this change.